addons/account/models/chart_template.py

```diff
@@ -933,6 +945,75 @@ def _generate_tax(self, company):
             'account_dict': todo_dict
         }
 
+# Tax Repartition Line Template
+
+class AccountTaxRepartitionLineTemplate(models.Model):
+    _name = "account.tax.repartition.line.template"
+    _description = "Tax Repartition Line Template"
+
+    factor_percent = fields.Float(string="%", required=True, help="Factor to apply on the account move lines generated from this repartition line, in percents")
+    repartition_type = fields.Selection(string="Based On", selection=[('base', 'Base'), ('tax', 'of tax')], required=True, default='tax', help="Base on which the factor will be applied.")
+    account_id = fields.Many2one(string="Account", comodel_name='account.account.template', help="Account on which to post the tax amount")
+    plus_tag_ids = fields.Many2many(string="Plus Tax Grids", relation='account_tax_repartition_plus_tags', comodel_name='account.tax.report.line', copy=True, help="Tag template whose '+' child will be assigned to move lines by this repartition line")
+    minus_tag_ids = fields.Many2many(string="Minus Tax Grids", relation='account_tax_repartition_minus_tags', comodel_name='account.tax.report.line', copy=True, help="Tag template whose '-' child will be assigned to move lines by this repartition line")
+    financial_tag_ids = fields.Many2many(string="Financial Tags", relation='account_tax_repartition_financial_tags', comodel_name='account.account.tag', copy=True, help="Additional tags that will be assigned by this repartition line for use in financial reports")
```

Comment:
rename field into tag_ids
move plus_tag_ids and minus_tag_ids below (dedicated comments blabla helper)
rename them into plus_report_line_id and minus_report_line_id
rename the xml_ids referencing the report grid lines to depict these aren't account.account.tag